cfn-resources/project/test/contract-testing/cfn-test-create-inputs.sh

```diff
@@ -59,3 +59,15 @@ jq --arg org "${org_id}" \
 	--arg team_id "$team_id" \
 	'.OrgId?|=$org | .Name?|=$name |.Profile?|=$profile | .ProjectApiKeys[0].Key?|=$key_id | .ProjectTeams[0].TeamId?|=$team_id' \
 	"test/inputs_1_update.template.json" >"inputs/inputs_1_update.json"
+
+jq --arg org "${org_id}" \
+	--arg name "${project_name}-tags" \
+	--arg profile "${profile}" \
+	'.OrgId?|=$org |.Name?|=$name |.Profile?|=$profile' \
+	"test/inputs_2_create.template.json" >"inputs/inputs_2_create.json"
+
+jq --arg org "${org_id}" \
```

Comment:
could we look into creating multiple test inputs via loop as in https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/search-deployment/test/cfn-test-create-inputs.sh#L50 ?

Reply:
Not sure if it makes sense.
The 1st template replaces more args: `$key_id` and `$team_id`
While the example you link to both only use `profile` and `clusterName`